src/pages/Models/_models.scss

```diff
@@ -15,6 +15,10 @@
     grid-template-columns: 1fr 0.6fr 0.75fr 1.15fr 1fr 0.5fr 0.65fr;
   }
 
+  .cloud-group tr {
+    grid-template-columns: 0.8fr 0.75fr 0.6fr 0.75fr 1.15fr 1fr 0.5fr 0.65fr;
```

Comment:
Lets give more spacing to the model name and less to the region 

<img width="1418" alt="Screen Shot 2020-01-14 at 12 39 27 PM" src="https://user-images.githubusercontent.com/532033/72371917-f29eab80-36ca-11ea-8ef9-ccffe677ace1.png">